server/lobbyconnection.py

```diff
@@ -1079,15 +1079,20 @@ async def abort_connection_if_banned(self):
                 return
 
             ban_expiry = data[ban.c.expires_at]
+            if now >= ban_expiry:
+                return
+
+            self._logger.debug('Aborting connection of banned user: %s, %s, %s',
+                               self.player.id, self.player.login, self.session)
+            raise_ban_error(ban_expiry - now, data[ban.c.reason])
+
+def raise_ban_error(ban_duration, ban_reason):
+    raise ClientError((f"You are banned from FAF {ban_duration_text(ban_duration)}.\n "
```

Comment:
Sorry, I thought I commented on this earlier, but I must have forgotten to click the "Add" button or something. The client actually expects these messages to be HTML
```suggestion
    raise ClientError((f"You are banned from FAF {ban_duration_text(ban_duration)}.</br> "
```
right now it looks like this (minus the white background which has something to do with linux):
![image](https://user-images.githubusercontent.com/17941539/91335536-10370600-e7c0-11ea-95b8-c42fd9914d4d.png)
The first one happens to line up correctly, but you can see there is no newline after the reason.

Reply:
Oh I keep clicking the wrong button I guess. I keep clicking "Add single comment" and not the green button which adds it to my review...

Reply:
Hopefully this is fixed now :)